src/sessions/base-session-handler.ts

```diff
@@ -108,6 +112,66 @@ export default abstract class BaseSessionHandler {
     throwSdkError.call(this.sdk, SdkErrorTypes.not_supported, `Audio mute not supported for sessionType ${session.sessionType}`, { params });
   }
 
+  // TODO: doc
+  async updateOutgoingMedia (session: IJingleSession, options: IUpdateOutgoingMedia): Promise<any> {
+    if (session.sessionType === SessionTypes.acdScreenShare) {
```

Comment:
So typically what I've been trying to do is keep type-specific behavior out of the base handler. I'm hoping this will help us avoid getting into a spaghetti situation like ewc. The way I'd handle this is in this base-session-handler, assume the ability to change outgoing media type. In the acd-screen-share handler, override this method to just do a log (which I think would just be an info-level log) and dont call the super's `updateOutgoingMedia.